risc0/cpp/risc0-runtime.h

```diff
@@ -0,0 +1,43 @@
+// Copyright Cartesi and individual authors (see AUTHORS)
+// SPDX-License-Identifier: LGPL-3.0-or-later
+//
+// This program is free software: you can redistribute it and/or modify it under
+// the terms of the GNU Lesser General Public License as published by the Free
+// Software Foundation, either version 3 of the License, or (at your option) any
+// later version.
+//
+// This program is distributed in the hope that it will be useful, but WITHOUT ANY
+// WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+// PARTICULAR PURPOSE. See the GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License along
+// with this program (see COPYING). If not, see <https://www.gnu.org/licenses/>.
+//
+
+#ifndef RISC0_RUNTIME_H
+#define RISC0_RUNTIME_H
+
+// Include third-party printf FIRST - provides printf/snprintf/vprintf/etc
+// These must be defined before any code that expects them
+#include "third-party/printf/printf.h"
```

Comment:
I wonder if we could simplify and remove the printf library from risc0 runtime, because putchar was completely disabled for state accessors used by ZKVM, so there is no way to print anymore. Could `printf` be a no-op , or are there other consequences? Or makes debugging harder?

If we need it, could we reuse the one already under `uarch/third-party/printf` ?